app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(10, 3)
+    assert sum(result) == 10, (
```

Comment:
```suggestion
    assert sum(split_integer(10, 3)) == 10, (
```

I think  no need create variable, if it uses  only once

Reply:
Please fix everywhere

Reply:
Yes sure, fixed it.
Thank you